apps/antalmanac/src/components/RightPane/CoursePane/CoursePaneRoot.tsx

```diff
@@ -71,7 +71,7 @@ class RightPane extends PureComponent {
 
     render() {
         return (
-            <>
+            <div style={{ padding: 8 }}>
```

Comment:
Not sure where the source of the issue is, but the GIF has been shifted up in the Search pane:

<img width="706" alt="Screenshot 2023-09-06 at 8 07 06 PM" src="https://github.com/icssc/AntAlmanac/assets/100006999/a5cadc94-095c-4878-8e5d-849d606d35f9">

Reply:
Is it supposed to be in the middle?

Reply:
Yeah, center of the pane. You can see the intended behavior on the Map pane